.github/scripts/install_dep.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -e
+
+echo "Update archives"
+
+sudo add-apt-repository ppa:longsleep/golang-backports
+sudo apt-get update
+
+echo "Install golang"
+sudo apt-get install golang-1.13
+export GO111MODULE='on'
+
+echo "Install operator sdk"
+curl -fsSL https://github.com/operator-framework/operator-sdk/releases/download/v0.17.0/operator-sdk-v0.17.0-x86_64-linux-gnu > operator-sdk
+chmod +x operator-sdk
+sudo mv operator-sdk /usr/local/bin/operator-sdk
+
+echo "Clean cache"
+sudo apt-get clean
+
+#Docker installation from google default repositories
```

Comment:
@EricWittmann Looks good ! However, the installation of docker and go-lang might not be required as the virtual runners already come preinstalled with Docker and Go-lang.

We can see the list of already installed Softwares [here](https://github.com/actions/virtual-environments/blob/master/images/linux/Ubuntu1804-README.md)

Reply:
OK thanks @riprasad .  @faamartinrh thoughts?  Do you want to try it without the install steps?

Reply:
I'm going to try it

Reply:
yeah it works, I didn't need to install docker and go-lang